packages/block-library/src/quote/deprecated.js

```diff
@@ -151,15 +240,22 @@ const v1 = {
 	migrate( attributes ) {
 		if ( attributes.style === 2 ) {
 			const { style, ...restAttributes } = attributes;
-			return migrateToQuoteV2( {
-				...restAttributes,
+			const [ v1Attributes, innerBlocks ] =
+				migrateToQuoteV2( restAttributes );
+			const v4Attributes = migrateTextAlign( v1Attributes );
+
+			return {
+				v4Attributes,
+				innerBlocks,
 				className: attributes.className
 					? attributes.className + ' is-style-large'
 					: 'is-style-large',
-			} );
+			};
```

Comment:
Probably best to stick to the normal tuple return here

Reply:
Also, the `className` attribute is migrated to v2 after attributes have been migrated to v4 already. This could cause something unexpected down the road.